### ISSUE TITLE
Running in HA ?

### ISSUE THREAD

**Issue by gedw99** [cortezaproject/corteza#772]
For the DB Layer i did not see any docs for High Availability.

Cockroach is probably the easiest to setup that way ?

**Comment by tjerman** [comment]
That’s correct; there were some plans and drafts for adding CockroachDB but nothing was set in stone yet
